components/eventing-controller/controllers/subscriptionv2/eventmesh/reconciler_internal_integration_test.go

```diff
@@ -231,17 +230,16 @@ func Test_replaceStatusCondition(t *testing.T) {
 		},
 	}
 
-	g := NewGomegaWithT(t)
 	r := Reconciler{}
 
 	for _, tt := range testCases {
 		t.Run(tt.name, func(t *testing.T) {
 			sub := tt.giveSubscription
 			condition := tt.giveCondition
 			statusChanged := r.replaceStatusCondition(sub, condition)
-			g.Expect(statusChanged).To(BeEquivalentTo(tt.wantStatusChanged))
-			g.Expect(sub.Status.Conditions).To(ContainElement(condition))
-			g.Expect(sub.Status.Ready).To(BeEquivalentTo(tt.wantReady))
+			require.Equal(t, tt.wantStatusChanged, statusChanged)
```

Comment:
(nit)
you can use `assert` instead of `require`, then the tests doesn't panic and checks the further assertions. In that case you have a better overview over the assertions.

Reply:
I have changed it to use assert.